lib/sm/client.rb

```diff
@@ -110,6 +111,8 @@ def connection
         conn.use :breakers
         conn.request :multipart
         conn.request :json
+        # conn.request :curl, ::Logger.new(STDOUT), :warn
```

Comment:
@aub it's just for debugging purposes, convenience really. MHV has been having some issues and I need to send them examples of what fails. Also, currently Faraday is doing something really weird client.move_message -- the output from faraday_curl works fine, but the actual request from faraday fails. So I'm downloading wireshark to intercept the request and see what's wrong.

Really this gem is meant more for dealing with annoying one-off scenarios where I need to send an email to MHV asking them why some request is failing. If we don't want to merge I can live with that though.